src/collectors/freebsd.plugin/freebsd_sysctl.c

```diff
@@ -24,6 +24,10 @@
 #include <netinet/udp.h>
 #include <netinet/udp_var.h>
 
+#define _COMMON_PLUGIN_NAME "freebsd.plugin"
+#define _COMMON_PLUGIN_MODULE_NAME "freebsd"
```

Comment:
Why is module name “freebsd”? It was a different value before your change? Can you please take a break and go through your changes yourself and ensure there are no obvious issues?

Reply:
We must have `_COMMON_PLUGIN_MODULE_NAME` to use headers, so I add a parameter to the function to ignore this option.